web/cookiesCTF/main.py

```diff
@@ -0,0 +1,122 @@
+from bottle import route, run, request, response, static_file, get, template
+
+
+@route('/')
+def hello():
+	f = open('./Pages/index.html')
+	return f.read()
+
+
+@get('/styles.css')
+def retCss():
+	return static_file('styles.css', root='./Pages')
+
+
+@get('/', method='POST')
+def checkFlag():
+	flag = request.forms.get('flag')
+	if flag == "PB{eat_my_cookies}" or flag == "PB{I'm_a_cookiebot}" or flag == "PB{Jake_don't_mind}" or flag == "PB{Jake_isn't_lonely}" or flag == 'PB{Y0tub3_v1d30s_ar3_fun}':
+		f = open('./Pages/Correct.html')
+		return f.read()
+	else:
+		f = open('./Pages/TryAgain.html')
+		return f.read()
+
+
+@route('/1')
+def task1():
+	response.set_cookie("is_admin", "no", path="/")
+	if request.get_cookie("is_admin") == "no":
+		f = open('./Pages/Task1/Task1GUEST.html')
+		return f.read()
+	elif request.get_cookie("is_admin") == "yes":
+		f = open('./Pages/Task1/Task1ADMIN.html')
+		return f.read()
```

Comment:
And what happens if the cookie is neither yes nor no? The server just doesn't send a response? Tell me if you have done this on purpose, and that's part of the task, but I assume you always want the server to send at least some response.

There are multiple ways to implement this, the simplest way I see is to return the Task1ADMIN.html page if the cookie is set to "yes", and return the guest page in any other case. By the way, I can see that you're already doing a similar thing in task 2. So why not do it here?